src/ipyvizzustory/storylib/story.py

```diff
@@ -132,22 +133,71 @@ def add_step(self, step: Step) -> None:
 class StorySize:
     """A class for representing the size of a presentation story."""
 
-    def __init__(self, width: Optional[str] = None, height: Optional[str] = None):
+    def __init__(
+        self,
+        width: Optional[Union[int, float, str]] = None,
+        height: Optional[Union[int, float, str]] = None,
+        aspect_ratio: Optional[Union[int, float, str]] = None,
+    ):
         """
         StorySize constructor.
 
         Args:
             width: The width of a presentation story.
             height: The height of a presentation story.
+            aspect_ratio: The aspect-ratio of a presentation story.
+
+        Raises:
+            ValueError: If width, height and aspect_ratio are set together.
         """
+
+        width = self._convert_to_pixel_or_return(width)
+        height = self._convert_to_pixel_or_return(height)
+
         self._width = width
         self._height = height
+        self._aspect_ratio = aspect_ratio
 
         self._style = ""
-        if any([width is not None, height is not None]):
-            width = "" if width is None else f"width: {width};"
-            height = "" if height is None else f"height: {height};"
-            self._style = f"vp.style.cssText = '{width}{height}'"
+        if all([width is not None, height is not None, aspect_ratio is not None]):
```

Comment:
`if None not in [width, height, aspect_ratio]:` ?

Reply:
fixed in [323bebd](https://github.com/vizzuhq/ipyvizzu-story/pull/75/commits/323bebd16454db182989946f6ef1440d1715c1d0)